src/gen_swift/templates/module.swift

```diff
@@ -73,9 +73,9 @@ class RNBreezSDK: RCTEventEmitter {
     {% endif -%}
     {%- endfor %}  
     @objc(setLogStream:reject:)
-    func setLogStream(_ resolve: @escaping RCTPromiseResolveBlock, reject: @escaping RCTPromiseRejectBlock) -> Void {
+    func setLogStream(_ filterLevel: LevelFilter = nil, resolve: @escaping RCTPromiseResolveBlock, reject: @escaping RCTPromiseRejectBlock) -> Void {
```

Comment:
React Native only likes basic types going over the bridge between native & javascript. So we should change this to an optional String, then convert it.
```suggestion
    func setLogStream(_ filterLevel: String?, resolve: @escaping RCTPromiseResolveBlock, reject: @escaping RCTPromiseRejectBlock) -> Void {
```